backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ReleaseSummary.java

```diff
@@ -109,6 +109,7 @@ private void setShortSummaryFields(Release document, Release copy) {
         copyField(document, copy, _Fields.RELEASE_DATE);
         copyField(document, copy, _Fields.SOURCE_CODE_DOWNLOADURL);
         copyField(document, copy, _Fields.BINARY_DOWNLOADURL);
+        copyField(document, copy, _Fields.EXTERNAL_IDS);
```

Comment:
No changes are needed in this file (ReleaseSummary). Kindly revert this change.

Reply:
done.